docs/ht_ww.md

```diff
@@ -0,0 +1,52 @@
+
+As with everything else, this too can be changed, and several plugins are avaliable.  Some work better than others.
+
+## List of OVOS WakeWord Plugins
+| Plugin                                 | Type     | Description                           |
+|----------------------------------------|----------|----------------------------|
+| [ovos-ww-plugin-precise-lite](https://github.com/OpenVoiceOS/ovos-ww-plugin-precise-lite) | Model | The most accurate plugin avaliable as it uses [pretrained models](https://github.com/OpenVoiceOS/precise-lite-models) and [community models](https://github.com/OpenVoiceOS/ovos-ww-community-dataset) are avaliable also |
+| [ovos-ww-plugin-openWakeWord](https://github.com/OpenVoiceOS/ovos-ww-plugin-openWakeWord) | Model | Uses [openWakeWord](https://www.github.com/dscripka/openwakeword) for detection |
+| [ovos-ww-plugin-vosk](https://github.com/OpenVoiceOS/ovos-ww-plugin-vosk) | Full Word | Uses full word detection from a loaded model. |
+| [ovos-ww-plugin-pocketsphinx](https://github.com/OpenVoiceOS/ovos-ww-plugin-pocketsphinx) | Phonomes | Probably the least accurate, but can be used on almost any device |
+| [ovos-ww-plugin-hotkeys](https://github.com/OpenVoiceOS/ovos-ww-plugin-hotkeys) | Model | Use an input from keyboard or button to emulate a wakeword being said.  Useful for privacy, but not so much for a smart speaker. |
+| [ovos-ww-plugin-snowboy](https://github.com/OpenVoiceOS/ovos-ww-plugin-snowboy) | Model | Uses [snowboy wakeword engine](https://github.com/seasalt-ai/snowboy) |
+| [ovos-ww-plugin-nyumaya](https://github.com/OpenVoiceOS/ovos-ww-plugin-nyumaya) | Model | WakeWord plugin using [Nyumaya](https://github.com/nyumaya) |
+
+## Configuration
+The configuration for wakewords are in the `"listener"` section of `mycroft.conf` and configuration of hotwords is in the `"hotwords"` section of the same file.
+
+This example will use the vosk plugin and change the wake word to "hey Ziggy".
+
+Add the following to your `~/.config/mycroft/mycroft.conf` file.
+```
+{
+    "listener": {
+        "wake_word": "hey_ziggy"
+    }
+    "hotwords": {
+        "hey_ziggy": {
+            "module": "ovos-ww-plugin-vosk",
+            "listen": true,
+            "active": true,
+            "sound": "snd/start_listening.wav",
+            "debug": false
+            "rule": "fuzzy",
+            "lang": "en",
+            "samples": [
+                "hey ziggy",
+                "hay ziggy"
+                ]
+        }
+    }
+}
+```
+
+The most important section is `"wake_word": "hey_ziggy"` in the `"listener"` section, which tells OVOS what the default wakeword should be, and in the `"hotwords"` section, `"active": true`, which tells the assistant to use the wakeword. If you want to disable a wakeword, you can set this to `false`. If enabling a wakeword, be sure to also set `"listen": true`.
```

Comment:
active: True is NOT necessary, the ww in listener handles this  (sets it to True UNLESS user set it to False)

you only need to set active: True if you want multiple wws, this is to enable the secondary ones

Reply:
Will fix, this was copied from the other file.